src/fs/rock/RockRebuild.cc

```diff
@@ -696,12 +697,16 @@ Rock::Rebuild::swanSong()
     storeRebuildComplete(&counts);
 }
 
-// TODO: After PR #691, revise callException() and failure() to supply errno
-// detail via SysErrorDetail and exception detail via ExceptionErrorDetail.
```

Comment:
Let's ignore that old TODO for now. It may have been based on an earlier version of (monumental!) error detailing effort that eventually resulted commit 83b053a or it could have been just a bad idea. We can finish this PR without that TODO.

Reply:
Ok, I removed this TODO.